### PR TITLE
Conference with more fields

### PR DESCRIPTION
**Pull request by jhthorsen**

### CODE REVIEW

File: lib/MCT.pm [hunk 1]
```diff
@@ -6,12 +6,15 @@ our $VERSION = '0.01';
...
+use MCT::Model::Countries;
 
 has pg => sub { Mojo::Pg->new(shift->config->{db}) };
 
 sub migrations {
   my $app = shift;
-  $app->pg->migrations->from_file($app->home->rel_file('mct.sql'));
+  $app->pg->migrations->from_string(
+    $app->build_controller->render_to_string(template => 'sql/migrations', format => 'sql')
+  );
```
Comment (jberger) [inline lib/MCT.pm]: now that we aren't interpolating all those countries, can we go back to a simple file rather than a template?

**Comment by jberger** [comment]
Beyond the comment above, the only question in this PR is, do we want to be moving closer to an ORM, which this is clearly doing? @tempire I would be curious about you opinions, @marcusramberg too if you are watching

File: lib/MCT.pm (hunk 1 above)
Comment (jhthorsen) [inline lib/MCT.pm]: I agree. Let's go back to a plain file.

**Comment by jhthorsen** [comment]
@jberger: We want to be non-blocking. The closest to an ORM I know that can do that is Mad::Mapper...

**Comment by jberger** [comment]
This is good work and I'm :+1: on the coding as long as @tempire is ok with the direction it takes the project.

**Comment by jberger** [comment]
well I guess this is necessary for sanity moving forward.

**Pull request merged**